masterlist.yaml

```
@@ -22689,16 +22689,28 @@ plugins:
         util: 'SSEEdit v4.0.4b'
 
   - name: 'The Great City of Solitude.esp'
-    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/22243/' ]
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/22243/'
+        name: 'The Great City of Solitude SSE'
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/72117/'
+        name:  'Rob''s Bug Fixes - TGC Solitude'
     after:
       - name: 'KvHarborside.esp'
         condition: 'file("TGCoS [PATCH - KvHarborside].esp") or file("TGCoS + Solitude Skyway SE [PATCH - KvHarborside].esp")'
     tag: [ C.Location ]
+    msg:
+      - <<: *patchUpdateAvailable
+        subs: [ '[Rob''s Bug Fixes - TGC Solitude](https://www.nexusmods.com/skyrimspecialedition/mods/72117/)' ]
+        condition: 'checksum("The Great City of Solitude.esp", BB0F093C)'
```

The `msg` key comes before `tag`:
```suggestion
    msg:
      - <<: *patchUpdateAvailable
        subs: [ '[Rob''s Bug Fixes - TGC Solitude](https://www.nexusmods.com/skyrimspecialedition/mods/72117/)' ]
        condition: 'checksum("The Great City of Solitude.esp", BB0F093C)'
    tag: [ C.Location ]
```